src/node/transaction.cpp

```diff
@@ -34,7 +34,8 @@ TransactionError BroadcastTransaction(NodeContext& node, const CTransactionRef t
     assert(node.peerman);
     assert(node.mempool);
     std::promise<void> promise;
-    uint256 hashTx = tx->GetHash();
+    uint256 txid = tx->GetHash();
+    uint256 wtxid = tx->GetWitnessHash();
```

Comment:
Very smol suggestion for ease of review - This improvement in the naming is in the "[mempol] Allow rebroadcast for same-txid-different-wtxid transactions" commit 14cf5dec17af9cc57a85dfe4910f30d8e17581d8, but would fit better in the subsequent [style] commit.

Reply:
I changed the name `hashTx` -> `txid` in the same commit as adding the `wtxid` variable. It seemed to make sense to me to have the variables named `txid`/`wtxid` from the start instead of `hashTx`/`wtxid`.

Not sure the ordering matters that much. I'm inclined to leave it as it is right now, but will happily reorder/split commits if that's helpful for other reviewers.